friends/src/components/Friend.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+export const Friend = ({ friend, editFriend, deleteFriend, history }) => {
+  return (
+    <div>
+      <div className="friend-header">
+        <h2>{friend.name}</h2>
+        <button onClick={() => editFriend(friend.id, history)}>
+          Edit Friend
+        </button>
+        <i
+          onClick={() => deleteFriend(history)}
```

Comment:
as of right now this doesn't work in my browser -- this might be because of how you declared your delete function to need history actually. you should only need to pass in an id for that endpoint (look at TK examples or Dustin's demo) like `/friends/${id}` so there's no need to pass history through here